src/Voron/Impl/LowLevelTransaction.cs

```diff
@@ -87,6 +87,26 @@ public void Reset()
 
 
         public event Action<IPagerLevelTransactionState> BeforeCommitFinalization;
+
+        public long TotalEncryptionBufferSize
+        {
+            get
+            {
+                var cryptoTransactionStates = ((IPagerLevelTransactionState)this).CryptoPagerTransactionState;
```

Comment:
Why do you need this cast?

Reply:
because somebody decided to implicitly implement this method and this is the only way to access it